lib/cartopy/_epsg.py

```diff
@@ -22,42 +22,20 @@
 from __future__ import (absolute_import, division, print_function)
 
 import cartopy.crs as ccrs
+from cartopy._proj4 import _PROJ4Projection
 import numpy as np
 import pyepsg
-import shapely.geometry as sgeom
 
 
-_GLOBE_PARAMS = {'datum': 'datum',
-                 'ellps': 'ellipse',
-                 'a': 'semimajor_axis',
-                 'b': 'semiminor_axis',
-                 'f': 'flattening',
-                 'rf': 'inverse_flattening',
-                 'towgs84': 'towgs84',
-                 'nadgrids': 'nadgrids'}
-
-
-class _EPSGProjection(ccrs.Projection):
+class _EPSGProjection(_PROJ4Projection):
```

Comment:
Seems highly likely that we will remove _EPSGProjection and replace it with a function that returns an appropriate proj4 string > a real cartopy.crc.CRS.

Reply:
To be clear: nothing for you to do here, just sharing my thoughts.